app/models/drawing.rb

```diff
@@ -2,6 +2,9 @@ class Drawing < ActiveRecord::Base
   enum status: %i(pending complete)
   enum gender: %i(not_specified female male other)
 
+  STAGES_OF_JOURNEY = ["At home", "In temporary shelter", "Awaiting transit", "On the move", "Arrived at destination"].freeze
```

Comment:
@krissy Is there any rules I don't know about the order these need to go in? I've always put constants at the very top of the class, but I don't know if that would be above/below enums